aries-core/src/js/components/layouts/EmptyState/EmptyState.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box, Heading, Paragraph } from 'grommet';
+
+export const EmptyState = ({
+  actions,
+  align,
+  description,
+  icon,
+  level,
+  title,
+}) => {
+  return (
```

Comment:
Would something like this be useful so that the EmptyState layout component has xlarge by default?

```suggestion
 if (!icon.size) {
   icon.size="xlarge"
 }

  return (
```

Reply:
Yes, I think that's a good idea.

Reply:
Done.